clusterloader2/testing/load/config.yaml

```diff
@@ -48,12 +48,34 @@
 {{$mediumDeploymentsPerNamespace := SubtractInt $mediumDeploymentsPerNamespace (IfThenElse $ENABLE_JOBS 1 0)}}
 {{$bigDeploymentsPerNamespace := SubtractInt $bigDeploymentsPerNamespace (IfThenElse $ENABLE_JOBS 1 0)}}
 
+# BEGIN scheduler-throughput section
+# Min number of pods per deployment to be used for measuring scheduler throughput
+# to get enough samples and accurate measurements in small clusters.
+{{$MIN_PODS_PER_DEPLOYMENT_TO_MEASURE_SCHEDULER_THROUGHPUT := 250}}
+# Number of namespaces with scheduler throughput deployments. This is also number
+# of scheduler throughput deployments as we create 1 such deployment per namespace.
+{{$schedulerThroughputNamespaces := 2}}
+{{$schedulerThroughputPodsPerDeployment := MaxInt $MIN_PODS_PER_DEPLOYMENT_TO_MEASURE_SCHEDULER_THROUGHPUT .Nodes}}
+{{$schedulerThroughputThreshold := DefaultParam .CL2_SCHEDULER_THROUGHPUT_THRESHOLD 0}}
+# END scheduler-throughput section
+
+# BEGIN pod-startup-latency section
```

Comment:
I would really like to get rid of it completely - we should provide SLO not only for some pods.
There are things that need to happen for that (I guess the biggest problem is throughput of a single node, but maybe this can be mitigated by switching to containerd given our recent numbers?)

That said, can you please add a TODO (and open an issue) to use pod-startup measured through the whole test [the measurement is already there, we just need to debug, fix and start enforcing it]